target_postgres/tests/test_standard_target.py

```diff
@@ -326,6 +326,42 @@ def test_large_int(postgres_target):
     singer_file_to_target(file_name, postgres_target)
 
 
+def test_anyof(postgres_config_no_ssl, engine):
+    """Test that anyOf is handled correctly"""
+    table_name = "commits"
+    file_name = f"{table_name}.singer"
+    singer_file_to_target(file_name, TargetPostgres(config=postgres_config_no_ssl))
+    with engine.connect() as connection:
+        result: sqlalchemy.engine.cursor.LegacyCursorResult = connection.execute(
+            f"SELECT column_name, data_type FROM information_schema.columns WHERE table_name = '{table_name}'"
+        )
+        breakpoint()
+        for row in result.all():
+            # {"type":"string"}
+            if row[0] == "id":
```

Comment:
Relying on the order here is a bit flimsy, if we could use the column names instead, I think this would be easier with using SQLAlchmey's meta (although pretty easy to do with sql too)

Reply:
Switched to using SQLAlchemy.